Simple-step-by-step-setup-instructions-for-non-Linux-users.md

```diff
@@ -0,0 +1,225 @@
+
+* Any modern machine with at least 2 GB per core.
+* VPS (_virtual private server_) recommended.
+* Unlimited bandwidth - current implementation consumes roughly 15GB/day.
+
+
+## Brief list of VPS poviders
+
+
+| URL           | Specs         | Bandwidth | Storage | Price/month  |
+| ------------- |:-------------:| :-------------:| :-----:| :-----:|
+| https://vultr.com | 2vCPU 4GB | 3 TB | 60GB | $20 |
+| https://try.digitalocean.com | 2vCPU 4GB| 4 TB | 80GB     |   $100 Free Trial |
+| https://www.hetzner.com/ | 2vCPU 4GB | 20TB  | 40TB    |    4,90€|
+| https://contabo.com/ | 4 CPU 8GB | Unlimited | 200 GB      |    4,99€ |
+| https://www.linode.com/ |2vCPU 4GB | 4TB | 80GB      |    $20 |
+| https://www.ovh.com/ | 2vCPU 7GB | ?  | 50GB    |    $26.40 |
+
+
+* Start by selecting a VPS provider, spin up a node selecting Ubuntu 18.04. The Community Dev Tree accepts other distributions, but for this guide we will focus on Ubuntu 18.04.
+
+* Once you’ve fired up your node, `ssh` into the server and perform typical maintenance (note, it may not look exactly the same on your machine)
+
+
+    sudo apt-get update
+
+![1](simple-setup-images/1.png)
+
+
+    sudo apt-get upgrade
+
+![2](simple-setup-images/2.png)
+
+* If this window shows up, select the first option
+
+![3](simple-setup-images/3.png)
+
+* We will now install git
+
+
+    sudo apt-get install -y git
+
+![4](simple-setup-images/4.png)
+
+* Clone Community Tree Graft Supernode testnet code from GitHub repository
+
+      git clone --recursive https://github.com/graft-community/graft-ng.git
+
+![5](simple-setup-images/5.png)
+
+* Enter the graft-ng directory
+
+
+    cd graft-ng
+
+![6](simple-setup-images/6.png)
+
+* Switch over to the alpha3 branch
+
+
+    git checkout alpha3
+
+![7](simple-setup-images/7.png)
+
+* Update submodules
+
+
+    git submodule update --init --recursive
+
+![8](simple-setup-images/8.png)
+
+* Install dependencies
+
+
+    sudo apt-get -y install build-essential cmake pkg-config libboost-all-dev libssl-dev autoconf automake check libpcre3-dev rapidjson-dev libreadline-dev
+
+![9](simple-setup-images/9.png)
+
+* Get back out of “graft-ng” directory, create a build directory, and enter it - this is where the actual Graft source code will be built.
+
+
+    cd
+    mkdir -p supernode
+    cd supernode
+
+![10](simple-setup-images/10.png)
+
+* Run “cmake” in this directory, using the source code
+
+
+    cmake -DENABLE_SYSLOG=ON $HOME/graft-ng
+
+![11](simple-setup-images/11.png)
+
+* Build the source code. This can take a while! (1h or more)
+
+
+    make
+
+![12](simple-setup-images/12.png)
+
+### Great! We’ve managed to install everything.
+You now have;
+
+* `graftnonded` - syncs the blockchain, has interactive commands
+* `graft-wallet-cli` - wallet management, has interactive commands
+* `graft_server` - RTA/supernode functions, non-interactive
+
+>`Graftnoded` and `graft-wallet-cli` both live inside the directory; `$Home/supernode/BUILD/bin/`
```

Comment:
This won't be the case on the `gc-alpha` branch; I improved some of the build structure, a consequence of which was to put them directly in the build directory rather than under `BUILD/bin`.